tests/unit/test_create_dataset.py

```diff
@@ -0,0 +1,349 @@
+from google.cloud.bigquery.dataset import Dataset, DatasetReference
```

Comment:
Need to have Apache Header on all code files

https://opensource.google/docs/releasing/preparing/#Apache-header

Reply:
On it. :)